### DISCUSSION TITLE
federation not work

### DISCUSSION THREAD

**Discussion by dormanze** [rabbitmq/rabbitmq-server#10781] [Other] [answered]
### Describe the bug

After I upgrade to the official release of rabbitmq [3.13.0](https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.13.0), the original federation plugin doesn't work.
I'm in the same environment, replace the version 3.11.9, the Federation plugin works fine.

### Reproduction steps

Installing RabbitMQ 3.13.0.
1. rabbitmqctl set_parameter federation-upstream test '{"uri":"amqps://user:paasword@7.241.62.145:31777","ack-mode":"on-confirm"}
2. rabbitmqctl set_policy exchange-default-federation ".*" '{"federation-upstream-set":"all"}' --priority 10 --apply-to exchanges

### Expected behavior

The federation plugin is working properly.

### Additional context

![image](https://github.com/rabbitmq/rabbitmq-server/assets/37854724/2c0ce4aa-d54c-4ff9-96be-45789d874f9c)

**Comment by mkuratczyk** [comment]
First of all, check the logs. There should be some information there.
Second, quite likely you also upgraded to Erlang 26 at the same time. Erlang 26 defaults to TLS peer verification. Given you're using TLS, quite likely it is a certificate validation problem. Older Erlang versions defaulted to no peer verification.
https://www.erlang.org/blog/otp-26-highlights/#ssl-safer-defaults

**Comment by lukebakken** [comment, marked as answer]
The following URI should fix the issue, but will NOT enable server X509 certificate verification:

```
amqps://user:paasword@7.241.62.145:31777?verify=verify_none
```

If you wish to enable server certificate verification, please see [the docs](https://www.rabbitmq.com/docs/uri-query-parameters#tls).

**Reply by dormanze** [reply]
Yes, I'm going to add this parameter. It's working now. Thank you both.
